examples/benchmark.jl

```diff
@@ -5,6 +5,9 @@ using OptimizationProblems
 using NLPModels
 using NLPModelsJuMP
 using AmplNLReader
+using DataFrames
+using Plots
+gr()
```

Comment:
Do we need to insist on `gr`? Is it the only one that supports `png()`?

Reply:
No, it just my usual backend. But I'll remove and it will use whichever backend is installed.